packages/runtime/src/errors/errors-list.ts

```diff
@@ -417,6 +417,12 @@ maximun uint128`,
 		title: "Clear state programs cannot issue inner transactions.",
 		description: "Clear state programs cannot issue inner transactions",
 	},
+	BYTES_REPLACE_ERROR: {
```

Comment:
we can add more details here and inform the user exactly what happened. You can pass arguments to the errors